components/datetime/src/options/components.rs

```diff
@@ -402,8 +400,6 @@ pub enum Month {
 // https://unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table
 //
 /// Options for displaying the current week for the `components::`[`Bag`].
```

Comment:
Suggestion: It would be nice to explain what "week" means here, as it could easily be confused with "weekday".

Reply:
I've changed the wording to "week number" and added/expanded examples.